src/status_im/multiaccounts/core.cljs

```diff
@@ -145,3 +159,45 @@
   (fx/merge cofx
             {::switch-theme theme}
             (multiaccounts.update/multiaccount-update :appearance theme {})))
+
+(defn clean-path [path]
+  (if path
+    (string/replace-first path #"file://" "")
+    (log/warn "[nativ-module] Empty path was provided")))
+
+(fx/defn save-profile-picture
+  {:events [::save-profile-picture]}
+  [cofx path ax ay bx by]
+  (let [key-uid (get-in cofx [:db :multiaccount :key-uid])]
+    (fx/merge cofx
+              {::json-rpc/call [{:method     "multiaccounts_storeIdentityImage"
+                                 :params     [key-uid (clean-path path) ax ay bx by]
+                                 ;; NOTE: In case of an error we can show a toast error
+                                 :on-success #(re-frame/dispatch [::update-local-picture %])}]}
+              (multiaccounts.update/optimistic :images [{:url  path
```

Comment:
Should be updated to the new format

Reply:
it's still an array for multiaccount, I haven't changed it, it's a map for contacts (I don't want to make too many changes for now).
Do you think is still an issue?

Reply:
No, I was thinking that format for all places was changed